improver_tests/psychrometric_calculations/test_WetBulbTemperature.py

```diff
@@ -102,19 +102,20 @@ class Test_WetBulbTemperature(IrisTest):
 
     def setUp(self):
         """Set up test input cubes and output data values."""
-        temperature = np.array([[185.0, 260.65, 338.15]], dtype=np.float32)
+        temperature = np.array([[185.0, 260.65, 273.15, 338.15]],
```

Comment:
Is there a reason that you have changed the tests when all you have changed are the doc strings?